pages/formpage.js

```diff
@@ -0,0 +1,63 @@
+import { useRouter } from "next/router";
+import { Container, Card, Header, Footer } from "../styles/styles";
+import Link from "next/link";
+
+export default function FormPage() {
+  const router = useRouter();
+
+  return (
+    <>
+      <Header>
+        <h1>Add a new trip to your list</h1>
+      </Header>
+      <Container>
+        <Card>
+          <form>
+            <div>
+              <label htmlFor="location">Location:</label>
+              <br />
+              <input
```

Comment:
Please buy an id for this one: id="location"

Reply:
Fixed